frontend/src/features/public-form/components/FormFields/SectionSidebar.tsx

```diff
@@ -20,7 +34,31 @@ export const SectionSidebar = (): JSX.Element => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [miniHeaderRef?.current?.clientHeight])
 
-  return (
+  return isMobile && isOpen && activeSectionId ? (
```

Comment:
nit: probably don't need to use ternary here, can be made more readible with an if clause:

```tsx
if (isMobile && isOpen && activeSectionId) {
  return <MobileSectionSidebar />
}

return <.... />
```